src/js/common/schemaform/actions.js

```diff
@@ -190,9 +206,14 @@ export function uploadFile(file, uiOptions, onProgress, onChange, onError) {
         const fileData = uiOptions.parseResponse(JSON.parse(body), file);
         onChange(fileData);
       } else {
+        let errorMessage = req.statusText;
+        if (req.status === 429) {
+          errorMessage = `You’ve reached the limit for the number of submissions we can accept at this time. Please try again in ${timeFromNow(moment.unix(parseInt(req.getResponseHeader('x-ratelimit-reset'), 10)))}.`;
```

Comment:
Do we typically provide seconds when instructing users to repeat actions, do we expect them to measure the time they wait in seconds? Could minutes potentially be more practical for how they make use of the message? Looping in @peggygannon .

Reply:
The message will display the largest unit it can for the message. If it's over 60 seconds, it'll use minutes, if it's over 60 minutes it'll say hours, etc.